nixos/modules/services/misc/nzbget.nix

```diff
@@ -4,7 +4,7 @@ with lib;
 
 let
   cfg = config.services.nzbget;
-  pkg = pkgs.nzbget;
+  pkg = cfg.package;
```

Comment:
Maybe we can get rid of the variable `pkg` altogether?

Reply:
Done